### PR TITLE
Fix aenetworks and ARD tests

### PR DESCRIPTION
**Pull request by adrianheine**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Read [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site)
- [x] Read [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) and adjusted the code to meet them
- [x] Covered the code with tests (note that PRs without tests will be REJECTED)
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

This updates and replaces a few failing tests and relaxes the ARD valid url regex.

### CODE REVIEW

File: youtube_dl/extractor/ard.py [hunk 1]
```diff
@@ -249,18 +249,18 @@ def _real_extract(self, url):
 
 
 class ARDIE(InfoExtractor):
-    _VALID_URL = r'(?P<mainurl>https?://(www\.)?daserste\.de/[^?#]+/videos(?:extern)?/(?P<display_id>[^/?#]+)-(?P<id>[0-9]+))\.html'
+    _VALID_URL = r'(?P<mainurl>https?://(?:www\.)?daserste\.de/[^?#]+/videos(?:extern)?/(?P<display_id>[^/?#]+)-(?:video)?(?P<id>[0-9]+))\.html'
```
Comment (dstftw) [inline youtube_dl/extractor/ard.py:252]:
```suggestion
    _VALID_URL = r'(?P<mainurl>https?://(?:www\.)?daserste\.de/[^?#]+/videos(?:extern)?/(?P<display_id>[^/?#]+)-(?:video-?)?(?P<id>[0-9]+))\.html'
```

File: youtube_dl/extractor/ard.py [hunk 2]
```diff
@@ -127,8 +127,8 @@ class ARDMediathekIE(ARDMediathekBaseIE):
         'info_dict': {
             'id': '44726822',
             'ext': 'mp4',
-            'title': 'Was ist die Kunst der Zukunft, liebe Anna McCarthy?',
-            'description': 'md5:4ada28b3e3b5df01647310e41f3a62f5',
+            'title': 'SÜDLICHT: Was ist die Kunst der Zukunft, liebe Anna McCarthy? | ARD Mediathek',
```
Comment (dstftw) [inline youtube_dl/extractor/ard.py:130]: Basically, there should not be suffixes like ` | ARD Mediathek` so that title extraction should be fixed.

**Review by dstftw — CHANGES_REQUESTED** [review]
Also split into two separate PRs.

File: youtube_dl/extractor/ard.py (hunk 2 above)
Comment (adrianheine) [reply youtube_dl/extractor/ard.py:130]: Thanks! I actually went with trying the OpenGraph attributes instead of changing the test.

**Comment by adrianheine** [comment]
Thanks for the review, I split the AENetworks change off as #27847.

**Pull request merged**